homework.md

```diff
@@ -4,38 +4,38 @@
 
```

Comment:
Hi Micheale, I finished reviewing your answers, All are excellent, good job my friend

Reply:
Thanks, FranklinPTorres for taking the time to review my answers.

Reply:
Hi Mecheale,
I completed reviewing your response. Everything is wonderful; keep up the fantastic job!